kitsune/questions/views.py

```diff
@@ -533,8 +536,19 @@ def aaq(request, product_key=None, category_key=None, step=1):
 
             return HttpResponseRedirect(path)
 
-        if has_subscriptions:
-            zendesk_form = ZendeskForm(data=request.POST or None, product=product)
+        # If product allows loginless support
+        # and not logged in, use CATEGORY_CHOICES_LOGINLESS
+        if has_subscriptions or is_loginless:
```

Comment:
this could be also written:

```python
if has_subscriptions:
    categories = CATEGORY_CHOICES
    if not request.user.is_authenticated:
        categories = CATEGORIES_CHOICES_LOGINLESS
```

Reply:
Agreed.